cli/foundation-models/system/inference/text-to-image/text-to-image-batch-endpoint.sh

```diff
@@ -0,0 +1,92 @@
+set -x
+# the commands in this file map to steps in this notebook: https://aka.ms/azureml-infer-batch-sdk-text-classification
+
+# script inputs
+registry_name="azureml"
+subscription_id="<SUBSCRIPTION_ID>"
+resource_group_name="<RESOURCE_GROUP>"
+workspace_name="<WORKSPACE_NAME>"
+
+# This is the model from system registry that needs to be deployed
+model_name="runwayml-stable-diffusion-v1-5"
+
+# Path to input .csv file
+inputs_dir="./batch_data"
+
+# Validate the existence of the model in the registry and get the latest version
+model_list=$(az ml model list --name ${model_name} --registry-name ${registry_name} 2>&1)
+if [[ ${model_list} == *"[]"* ]]; then
```

Comment:
I guess, we don't need this. We are checking model below and also retreiving latest model there.